### PR TITLE
Replace travis with github actions

### PR DESCRIPTION
**Pull request by luca-rath**

### CODE REVIEW

**Comment by luca-rath** [comment]
CI results can be seen here https://github.com/luca-rath/MassiveBuildBundle/pull/1

**Review by alexander-schranz — APPROVED** [review]

File: .gitattributes [hunk 1]
```diff
@@ -1,5 +1,4 @@
 .gitattributes export-ignore
 .gitignore export-ignore
-.travis.yml export-ignore
```
Comment (alexander-schranz) [inline .gitattributes:3]: `.github` missing here

Comment (alexander-schranz) [inline .gitattributes:3]:
```suggestion
/.github export-ignore
```

File: .gitattributes [hunk 2]
```diff
@@ -1,5 +1,4 @@
 .gitattributes export-ignore
 .gitignore export-ignore
```
Comment (alexander-schranz) [inline .gitattributes:2]:
```suggestion
.gitignore export-ignore
/.github export-ignore
```

**Pull request merged**